src/test/java/ReaderWriterTest.java

```diff
@@ -0,0 +1,91 @@
+package test;
+
+import static org.junit.Assert.assertEquals;
+
+import IO.DbReader;
+import IO.DbWriter;
+import data.Database;
+import data.DatabaseQuery;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.Reader;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+/* Validate that a .txt file can be properly be stored written and read from database.  */
+public final class ReaderWriterTest {
+
+  @Test
+  public void validateDatabaseName() throws IOException {
+    System.out.println("validateDatabaseName()");
+    Database database = new Database("test");
+    /* Validate that place where data is stored is named test. */
+    System.out.println("test = " + database.getDatabaseName());
+    Assert.assertEquals(database.getDatabaseName(), "test");
+  }
+
+  @Test
+  public void validateWriterRunId() throws IOException {
+    System.out.println("validateWriterRunId()");
+    Database database = new Database("test");
+    /* Validate that writer's runId is test. */
+    DbWriter writer = new DbWriter(database, "test", "writingIn");
```

Comment:
Give an explicit name of runid and type, otherwise it will be confused with other parameters name.
e.g. new DbWriter(database, "run_id_a", "Pose");